src/sidebar/components/svg-icon.js

```diff
@@ -66,12 +66,17 @@ function SvgIcon({ name, className = '', inline = false }) {
   const element = useRef();
   useLayoutEffect(() => {
     const svg = element.current.querySelector('svg');
+    const svgTitle = svg.querySelector('title');
     svg.setAttribute('class', className);
+    if (title && svgTitle) {
```

Comment:
Don't worry, I'll comment this/clean it up in any "real" version. 

Note that the `title` element is only modified if a `title` element already exists in the `SVG`. Adding a `title` element dynamically to the DOM (were it missing) is certainly possible, _but_ with an odd side effect that tooltips won't show up for dynamically-added SVG `title`s, which defeats the entire point here. Modifying an existing `title` element works fine, though. Shrug.

Reply:
Hi Lyza - I think it would be simpler and safer to add the title to the wrapping `<span>` as you initially suggested on Slack, then just remove any `<title>` attribute which is already in the SVG file.

There is a security hazard here in that `title` is not sanitized before being applied via `innerHTML`. If user-controlled data found its way into `title` that could be used to add scripts via [event attributes](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/Events).